### ISSUE TITLE
Browser hangs after update 6.6.95

### ISSUE THREAD

**Issue by lukas-derksen** [Templarian/MaterialDesign#6519]
### Where are you experiencing this bug?

@mdi/font

### What is the issue you are experiencing?

As already mentioned in the [Webfont repository issues](https://github.com/Templarian/MaterialDesign-Webfont/issues/84), the browser hangs when trying to load the icons since the latest update. At first, I thought the CDN was to blame, but after locally installing the font, the problem still occurred. Downgrading to the previous version resolved the issue, so one of the updated files is to blame. 

At first glance, I don't see anything wrong with the CSS file, so I tend to think one (or more) of the font files are to blame, or possibly a mismatched reference in the CSS file to the font. I'll see if I can narrow it down, but I've got no particular in-depth experience with web fonts.

**Comment by Agoulaouic** [comment]
Same thing here, especially visible on Firefox.
I tried to install locally too, the problem still occured.
Reproduced by opening the preview.html file contained in the node_modules/@mdi\font folder in a new private window.

**Comment by Star-Best** [comment]
I can confirm the same issue. Temp downgrading the version fixed it.

**Comment by goyney** [comment]
Can you please confirm if the issue happens when visiting the official preview: https://pictogrammers.github.io/@mdi/font/6.6.95/

Can you also please provide more than just browser names?
- Browser version? 
- OS?
- Have you disabled all add-ons running the browser and tried again?

Thanks.

**Comment by irongomme** [comment]
The same for me ... I can confirm it for windows 8.1, with every chromium based browser (chrome, edge, etc...)

**Comment by DeadActive** [comment]
Issue happens also when I visit preview page on Chromium, Chrome and Firefox latest versions on Windows 10 without any add-ons. There is no issue on Linux/Unix systems.

**Comment by amurzeau** [comment]
Browser: 98.0.1
OS: Windows 10 21H1
When I middle click to load a new tab with https://pictogrammers.github.io/@mdi/font/6.6.95/, it loads fine without freeze
When I click on that tab to switch to it, firefox freeze for around 20s (on a ryzen 3600 cpu).

The stacktrace of the busy thread shows that firefox seems busy in DirectWrite calls to render the font.
`dwrite!DWriteFactory::CreateFontFace` is called multiple times, but I don't know how many.
```
dwrite!LigatureSubstitutionSubtableHelper::ReadLookupCoverage+0x1ed
dwrite!ReadLookupCoverageRanges+0x1e7
dwrite!FontFeatureCoverageRegionBuilder::ReadTable+0x125
dwrite!OpenTypeFontFaceBuilder::ReadFontFeatureCoverage+0x150
dwrite!OpenTypeFontFaceBuilder::ReadFontFaceExtended+0x9c
dwrite!FontFaceRegionBuilder::WriteRegionInternal+0x38
dwrite!FontFaceRegionBuilder::WriteRegion+0x13
dwrite!FontFaceExtendedConstructionTask::AddElementData+0x69
dwrite!CacheWriter::AddElement+0x72
dwrite!IBaseCacheContextImpl::AddElementInternal+0x5e
dwrite!DWriteFactory::AddElement+0x15
dwrite!ElementTaskList::ExecuteTask+0x40
dwrite!IBaseCacheContextImpl::GetElementData+0xf9
dwrite!FontFace::FontFace+0x2fd
dwrite!DWriteFontFace::DWriteFontFace+0x35
dwrite!DWriteFontFace::Create+0xfc
dwrite!DWriteFactory::CreateFontFace+0x101
xul!winapi::um::dwrite::IDWriteFactory::CreateFontFace+0x2b
xul!dwrote::font_file::FontFile::create_face+0x8a
xul!webrender::platform::windows::font::FontContext::add_raw_font+0xd4
xul!webrender::glyph_rasterizer::impl$20::add_font+0xac9
xul!webrender::glyph_rasterizer::impl$19::add_font::closure$0+0xad2
xul!webrender::glyph_rasterizer::impl$18::async_for_each::closure$0+0xcd1
xul!core::panic::unwind_safe::impl$23::call_once+0xd2f
xul!std::panicking::try::do_call+0xd2f
xul!std::panicking::try+0xd2f
xul!std::panic::catch_unwind+0xd2f
xul!rayon_core::unwind::halt_unwinding+0xd2f
xul!rayon_core::spawn::spawn_job::closure$0+0xd2f
xul!rayon_core::job::impl$6::execute<rayon_core::spawn::spawn_job::closure$0>+0xdc6
xul!rayon_core::registry::WorkerThread::wait_until_cold+0x573
xul!rayon_core::registry::WorkerThread::wait_until+0x20
xul!rayon_core::registry::main_loop+0x415
xul!rayon_core::registry::ThreadBuilder::run+0x415
xul!rayon_core::registry::impl$2::spawn::closure$0+0x46c
xul!std::sys_common::backtrace::__rust_begin_short_backtrace<rayon_core::registry::impl$2::spawn::closure$0,tuple$<> >+0x48b
xul!std::thread::impl$0::spawn_unchecked::closure$1::closure$0+0x13
xul!core::panic::unwind_safe::impl$23::call_once+0x13
xul!std::panicking::try::do_call+0x13
xul!std::panicking::try+0x13
xul!std::panic::catch_unwind+0x13
xul!std::thread::impl$0::spawn_unchecked::closure$1+0x4e
xul!core::ops::function::FnOnce::call_once<std::thread::impl$0::spawn_unchecked::closure$1,tuple$<> >+0x5b
xul!alloc::boxed::impl$44::call_once+0xb
xul!alloc::boxed::impl$44::call_once+0x16
xul!std::sys::windows::thread::impl$0::new::thread_start+0x4c
KERNEL32!BaseThreadInitThunk+0x14
mozglue!mozilla::interceptor::FuncHook<mozilla::interceptor::WindowsDllInterceptor<mozilla::interceptor::VMSharingPolicyShared>,void (*)(int, void *, void *)>::operator()+0x15
mozglue!patched_BaseThreadInitThunk+0x28
ntdll!RtlUserThreadStart+0x21
```

**Comment by accrisoftteam** [comment]
Reproduced on OS:
Windows 8
Windows 10
Windows 11

Reproduced on Browsers:
Chrome latest
Firefox latest

Reproduced on websites:
https://vuetifyjs.com/en/components/alerts/

NOT Reproduced on Browsers:
Chromium Edge latest

NOT Reproduced on OS:
Linux
Mac


============
Expected behavior

1. https://vuetifyjs.com/en/components/alerts/
2. Click around to other UI Components on the left navigation
3. Notice the browser **never** freezes


============
Actual behavior

1. https://vuetifyjs.com/en/components/alerts/
2. Click around to other UI Components on the left navigation
3. Notice the browser **does** freeze

**Comment by Templarian** [comment]
Latest has been set back to `6.5.95` until I can figure out what version of `webfont` causes this hiccup.

**Comment by jordanbrowneb** [comment]
@Templarian do you know if the "latest" tag rollback will propagate to CDNs like jsdelivr? The slowdown is causing issues for our apps as well.

**Comment by jendib** [comment]
@jordanbrowneb You just need to pin a version using this link: <https://cdn.jsdelivr.net/npm/@mdi/font@6.5.95/css/materialdesignicons.min.css>
Don't use "latest" in general, it's bad practice.

**Comment by goyney** [comment]
While we try to resolve the underlying issue that caused the problem, this would be a good time (and place) to remind people that they really should stop using the webfont. Especially... _especially_ using a "latest" tag. Our library grows by at least 100 icons every release, which means the assets all of your users have pull down grows pretty much every month. And I can guarantee you, your application isn't using all 6700 icons; you don't need them all.

Here's our guide to webfont alternatives: https://dev.materialdesignicons.com/guide/webfont-alternatives

And a blog post I wrote about this topic last year: https://www.irigoyen.dev/blog/2021/02/17/stop-using-icon-fonts/

Once we resolve the issue and get a fix up, we'll update this thread. But in the meantime, I encourage everyone to check out those thinks above.

**Comment by Templarian** [comment]
Publishing 6.6.96 now. https://pictogrammers.github.io/@mdi/font/6.6.96/

**Comment by Templarian** [comment]
All packages were released earlier for `6.6.96`. Issue should be resolved.

We'll be looking into the issue and try to give the `webfont` more details they can use to debug what might be going wrong.
